task/helm-render-manifests-from-repo/0.1/helm-render-manifests-from-repo.yaml

```diff
@@ -0,0 +1,85 @@
+# This Task will do a helm upgrade based on the given helm repo and chart
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: helm-render-manifests-from-repo
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/categories: Deployment
+    tekton.dev/pipelines.minVersion: "0.12.1"
```

Comment:
`tekton.dev/v1` was introduced in `v0.50.0`, can we fix the version here as well?